streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -666,12 +666,13 @@ public Long copy(Application appParam) {
         newApp.setProjectId(oldApp.getProjectId());
         newApp.setModule(oldApp.getModule());
         newApp.setDefaultModeIngress(oldApp.getDefaultModeIngress());
-
         newApp.setUserId(commonService.getUserId());
         newApp.setState(FlinkAppState.ADDED.getValue());
         newApp.setLaunch(LaunchState.NEED_LAUNCH.get());
         newApp.setOptionState(OptionState.NONE.getValue());
         newApp.setCreateTime(new Date());
+        Object yarnSessionClusterId = oldApp.getHotParamsMap().get(ConfigConst.KEY_YARN_APP_ID());
+        newApp.setYarnSessionClusterId(yarnSessionClusterId == null ? null : yarnSessionClusterId.toString());
```

Comment:
`YarnSessionClusterId` is transient, not save to table, juse add: `newApp.setHotParams(oldApp.getHotParams());`